### ISSUE TITLE
Dark mode doesn't seem to work in 114.

### ISSUE THREAD

**Issue by p7r0x7** [Alex313031/thorium#184]
**System Details**
 - OS: Arch Linux
 - DE/WM: AwesomeWM + KDE on Xorg in light mode.
 - Thorium Version 114.0.5735.134

**Problem**
Dark mode flag no worky no more.

**Comment by Alex313031** [comment]
@p7r0x7 Can you explain more and provide screenshots?

**Comment by p7r0x7** [comment]
The issue seems to have resolved itself, meaning it was likely a KDE update regression or something; music.apple.com is the only site that still exhibits the issue of no longer rendering in dark mode.

**Comment by p7r0x7** [comment]
Though:

<img width="1280" alt="image" src="https://github.com/Alex313031/thorium/assets/54943589/f3c15a06-3e0d-477b-b020-402208cf7248">

**Comment by p7r0x7** [comment]
Sites with manually set dark mode work, but others don't:

<img width="1280" alt="image" src="https://github.com/Alex313031/thorium/assets/54943589/4fcdc719-5640-4a7b-a1a7-a3aba289046a">

<img width="1280" alt="image" src="https://github.com/Alex313031/thorium/assets/54943589/bd4be840-c18a-45d6-94e3-9f921deaf2cd">

**Comment by p7r0x7** [comment]
<img width="1280" alt="image" src="https://github.com/Alex313031/thorium/assets/54943589/b529d7b3-1bf7-40ff-b101-58a4a0e129b2">

**Comment by p7r0x7** [comment]
YouTube is another notable one:

<img width="1280" alt="image" src="https://github.com/Alex313031/thorium/assets/54943589/a23743b7-94cc-4a47-9e23-58541066ccb6">

On another note, it seems like v114 fixed the hwdec issue for youtube.com I'd been having since I started using Thorium a couple releases ago. Thanks!

**Comment by gz83** [comment]
Regarding the internal pages of the Thorium browser (pages starting with chrome://), the dark mode cannot be applied. It may be a bug in the upstream version, or it may be intentional by Google.

Regarding external pages, please check if there are any settings related to dark mode and make sure that there is no problem with the related settings.

@p7r0x7

**Comment by lfitt** [comment]
I can confirm dark mode is broken in "Version 117.0.5938.157 (Official Build) stable, built on Linux (64-bit)" still.

Debian Testing / Cinnamon / GTK / Dark Mode.

I tested in vanilla Chromium "Version 114.0.5735.198 (Official Build) built on Debian 12.0, running on Debian 12.1 (64-bit)" 
and it exhibits the same behavior, so it's probably an "upstream" thing. 

it's as though it's aware the DE wants dark mode (it itself is largely in dark mode) it's just not passing the hints through to the website like firefox does to actually make dark mode happen. 

I have also changed the settings mentioned at the top of this issue, it has made no change.
